sdk/storage/AzureStorageBlob/Source/Data/BlockTransfer+CoreDataProperties.swift

```diff
@@ -29,13 +29,13 @@ import CoreData
 import Foundation
 
 extension BlockTransfer {
-    @nonobjc internal class func fetchRequest() -> NSFetchRequest<BlockTransfer> {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<BlockTransfer> {
```

Comment:
Same question about promoting these to public. It doesn't matter as much here because BlockTransfer is internal. I'm guessing these changes just happened automatically from the code generator and weren't intentional.

Reply:
Fixed (yes, code gen)